main.css

```diff
@@ -0,0 +1,107 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 92px;
+}
+
+.header-container > nav {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.nav-item1 {
+  flex-grow: 1;
+}
+
+.nav-item2 {
+  flex-grow: 8;
+}
+
+.nav-item3 {
+  flex-grow: 1;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/header.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  width: 375px;
+  height: 604px;
+  border-radius: 0 0 0 100px;
+  border-bottom: 2px solid #cfd8dc;
+}
+
+.intro {
+  width: 270px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 50px;
```

Comment:
- [ ] Great job so far, there font you used is a bit different than the one on the design please fix that if it's a google font make sure you import it before assigning it to your font family.

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/158662870-42e891c6-cf41-4f12-bf75-4bbd4b011c74.png)

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/158662964-2dafd2da-223e-477c-9542-122389ba7ae8.png)